test/tools/unified-spec-runner/schema.ts

```diff
@@ -20,9 +20,10 @@ export interface UnifiedSuite {
   description: string;
   schemaVersion: string;
   runOnRequirements?: RunOnRequirement[];
-  createEntities?: [EntityDescription, ...EntityDescription[]];
-  initialData?: [CollectionData, ...CollectionData[]];
-  tests: [Test, ...Test[]];
+  createEntities?: EntityDescription[];
+  /** Data inserted before **all tests */
+  initialData?: CollectionData[];
+  tests: Test[];
```

Comment:
I think the previous typings were trying to assert that the arrays have to have at least one member, was that incorrect?

Reply:
Yea can confirm, I generated the definitions from the schema and the schema has an enforcement of "at least 1" which caused this `[type, ...type[]]` to be generated, but it has more friction in practice that doesn't help us. I think I ran into issues with array methods like `.map` not typing easily.

Reply:
sounds good to me